org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -318,3 +318,44 @@ $button-border-radius: 2px;
     }
   }
 }
+
+%btn-primary {
```

Comment:
Why are you redefining `btn-primary`? Isn't it already available, given that this partial is importing it from the theme extension?
```scss
@import '../../../org.civicrm.bootstrap/scss/vendor/bootstrap/buttons';
 @import '../bootstrap/overrides/style/buttons';
```
(same goes with `%btn-default` below)